### ISSUE TITLE
SlackInteractivityHandler does not handle view_submission payloads from modals

### ISSUE THREAD

**Issue by robcoward** [hanchchch/nestjs-slack-listener#240]
The handleSingleInteractivity() function expects the payload to include `payload.actions[]` but in a `view_submission` payload (see https://api.slack.com/reference/interaction-payloads/views#view_submission ) sent when submitting a modal form, there is no list of actions in the payload so the code errors with:
```
[Nest] 76  - 07/04/2023, 6:08:45 PM   ERROR [ExceptionsHandler] Cannot read properties of undefined (reading 'filter')
TypeError: Cannot read properties of undefined (reading 'filter')
    at SlackHandler.<anonymous> (/usr/src/app/node_modules/nestjs-slack-listener/dist/slack/slack-handler.service.js:59:37)
```

Perhaps the code could be changed to be:
```
    if (actionId) {
      if (
        payload.actions &&
        payload.actions.filter((action) => action.action_id == actionId)
          .length == 0
      ) {
        return;
      }
    }
```
In fact the slack documentation seems to suggest using the `callback_id` field on a modal view to correctly handle returned `view_submission` payloads, so perhaps also adding:
```
    if (actionId) {
      if (
        payload.actions &&
        payload.actions.filter((action) => action.action_id == actionId)
          .length == 0
      ) {
        return;
      }
      if (
        payload.view &&
        payload.view.callback_id != actionId
      ) {
        return;
      }
    }
```

**Comment by hanchchch** [comment]
Hi! Thank you very much for reporting! And the `callback_id` idea perfectly makes sense. I'll fix it right away.

**Comment by hanchchch** [comment]
Fixed at the version `1.1.2`
